packages/venia-ui/lib/components/MiniCart/miniCart.css

```diff
@@ -1,79 +1,79 @@
 .root {
-    /* The root aside element takes up the whole screen. */
-    position: fixed;
-    left: 0;
-    top: 0;
     height: 100%;
-    width: 100%;
-
-    /* It is hidden by default. */
+    left: 0;
     opacity: 0;
-    visibility: hidden;
-
-    /* It animates to being closed. */
+    position: fixed;
+    top: 0;
     transition-duration: 192ms;
-    transition-timing-function: var(--venia-global-anim-out);
     transition-property: opacity, visibility;
-
-    /* It sits over all background content. */
+    transition-timing-function: var(--venia-global-anim-out);
+    visibility: hidden;
+    width: 100%;
     z-index: 3;
 }
 
 .root_open {
     composes: root;
-
     opacity: 1;
-    visibility: visible;
-
-    /* It animates to being open. */
     transition-duration: 224ms;
     transition-timing-function: var(--venia-global-anim-in);
-}
-
-.body {
-    overflow-y: scroll;
-    padding: 1.5rem 1rem 1.5rem 0rem;
+    visibility: visible;
 }
 
 .contents {
     background-color: rgb(var(--venia-global-color-background));
     border-radius: 1px;
-    box-shadow: 1px 1px 5px rgb(var(--venia-global-color-gray-darker));
+    box-shadow: 0 1px 6px rgb(var(--venia-global-color-gray-900) / 0.2);
```

Comment:
Dividing by `0.2` seems strange to me - should we have or could we use a different global gray?